### PR TITLE
Update .travis.yml g++ compiler

### PR DESCRIPTION
**Pull request by codydaig**

### CODE REVIEW

**Comment by lirantal** [comment]
@codydaig still not building. Possibly an npm hiccup?

**Comment by codydaig** [comment]
@lirantal It's producing the same errors as #237 over the course of several days. It's not the npm version either because the npm version varies through the node versions and they are all failing. I'll have to investigate this deeper as I'm not sure what's going on.

**Comment by lirantal** [comment]
Ok, glad you're on it pal.

File: .travis.yml [hunk 1]
```diff
@@ -11,12 +12,20 @@ os:
 matrix:
   allow_failures:
     - node_js: 5
```
Comment (ilanbiala) [inline .travis.yml]: Don't think we need to allow Node 5 to fail, considering we don't let it fail in https://github.com/meanjs/mean/blob/master/.travis.yml anymore

**Pull request closed without merging**